contracts/validators/CredentialAtomicQuerySigValidator.sol

```diff
@@ -79,4 +91,14 @@ contract CredentialAtomicQuerySigValidator is CredentialAtomicQueryValidator {
 
         return params;
     }
+
+    function _getSpecialInputPairs(
```

Comment:
could be moved to CredentialAtomicQueryValidator since userID and timestamp have same indexes for sig and mtp